src/app/qgsmaptooladdregularpolygon.h

```diff
@@ -0,0 +1,49 @@
+#ifndef QGSMAPTOOLADDREGULARPOLYGON_H
```

Comment:
The header (copyright, email, ...) seems to be lacking for this file.

Reply:
done